pygeo/DVConstraints.py

```diff
@@ -3133,6 +3484,336 @@ def writeTecplot(self, handle):
         for i in range(len(self.coords)//2):
             handle.write('%d %d\n'% (2*i+1, 2*i+2))
 
+class TriangulatedSurfaceConstraint(GeometricConstraint):
+    """
+    This class is used to enclose a triangulated object inside an
+    aerodynamic surface.
+    """
+
+    def __init__(self, name, surface_1, surface_1_name, DVGeo1, surface_2, surface_2_name, DVGeo2, scale, addToPyOpt,
```

Comment:
Can you  add doc strings for the inputs

Reply:
The init method is thinly wrapped by the addTriangulatedSurfaceConstraint method which has a comprehensive docstring. The other class __init__ methods in DVConstraints.py don't have docstrings. OK to skip?